### ISSUE TITLE
Crash when launching

### ISSUE THREAD

**Issue by sargonas** [dustinblackman/Championify#10]
I've come across an odd issue. On one of my computers, when I run it from the desktop (which is how I run it on my other two) it says it can't find my LoL folder and to run it from inside the install path. I moved it to /Riot Games/League and ran it again, and then the following error appears before closing itself:

[09:42:57] | Your version of Championify is up to date!
undefined:0

TypeError: Cannot read property 'replace' of undefined
    at <anonymous>
    at Math.Enclose.+c+:+\+t+h+e+b+o+x+\+C+h+a+m+p+\+n+o+d+e+_+m+o+d+u+l+e+s+\+g
+l+o+b+\+n+o+d+e+_+m+o+d+u+l+e+s+\+o+n+c+e+\+o+n+c+e+.+j+s.f (<anonymous>)
    at Glob.<anonymous> (<anonymous>)
    at Glob.emit (events.js:107:17)
    at Glob.Math.Enclose.+c+:+\+t+h+e+b+o+x+\+C+h+a+m+p+\+n+o+d+e+_+m+o+d+u+l+e+
s+\+g+l+o+b+\+g+l+o+b+.+j+s.Glob._finish (<anonymous>)
    at done (<anonymous>)
    at Glob.Math.Enclose.+c+:+\+t+h+e+b+o+x+\+C+h+a+m+p+\+n+o+d+e+_+m+o+d+u+l+e+
s+\+g+l+o+b+\+g+l+o+b+.+j+s.Glob._processReaddir2 (<anonymous>)
    at <anonymous>
    at RES (<anonymous>)
    at Math.Enclose.+c+:+\+t+h+e+b+o+x+\+C+h+a+m+p+\+n+o+d+e+_+m+o+d+u+l+e+s+\+g
+l+o+b+\+n+o+d+e+_+m+o+d+u+l+e+s+\+o+n+c+e+\+o+n+c+e+.+j+s.f (<anonymous>)

**Comment by dustinblackman** [comment]
Odd, that seems to be something with EncloseJS.
Thanks for the report, though it won't be an issue after the end of this week as I should have the GUI ready to roll.

**Comment by sargonas** [comment]
totally pointless now since the GUI is out and such, but just an FYI I learned this was because my exe was titled lollauncher.exe and not lol.launcher.exe (no first period). This was due to a side effect of me swapping around between production and dev builds of league, and should never happen to a normal person.

TL;DR - super *super* edge case that was my own doing!
